PrimeNo.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+public class PrimeNo {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a number to check if it is prime: ");
+        int num = scanner.nextInt();
+        
+        // You can change this number to test other values
+        boolean isPrime = true;
+
+        if (num <= 1) {
+            isPrime = false;
+        } else {
+            for (int i = 2; i <= Math.sqrt(num); i++) {
```

Comment:
Add a comment above the for loop explaining the need of the loop

Reply:
ok